blueprints/serializer-test/qunit-rfc-232-files/tests/unit/__path__/__test__.js

```diff
@@ -0,0 +1,24 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { run } from '@ember/runloop';
+
+module('<%= friendlyTestDescription %>', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
+    let store = this.owner.lookup('service:store');
+    let serializer = this.owner.factoryFor('serializer:<%= dasherizedModuleName %>').create({ store });
+
+    assert.ok(serializer);
+  });
+
+  test('it serializes records', function(assert) {
+    let store = this.owner.lookup('service:store');
+    let record = run(() => store.createRecord('<%= dasherizedModuleName %>', {}));
+
+    let serializedRecord = record.serialize();
+
+    assert.ok(serializedRecord);
```

Comment:
This should probably use some sort of `assert.deepEqual`

Reply:
Fine with whatever -- I simply copied the old ones https://github.com/emberjs/data/blob/master/blueprints/serializer-test/qunit-files/tests/unit/__path__/__test__.js#L14

Reply:
I was playing around with adding `assert.deepEqual({}, serializedRecord);` in my repo, and realized that if you've already implemented the model for which a serializer is being generated, at the moment we can't generate a test case that will pass when generated + when using the equality check.

I was under the impression that generated test cases should always pass -- do we want to continue to have this behavior?